lib/v2/ruijuzi/radar.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'rui.juzi.bot': {
```

Comment:
```suggestion
    'juzi.bot': {
```
The object can only match the domain name without a hostname. Use an inner key to match a hostname